site/content/docs/traces-open-telemetry/importing-traces/troubleshooting-missing-spans.md

```diff
@@ -0,0 +1,115 @@
+---
+title: Troubleshooting missing or orphan spans
+weight: 20
+menu:
+  platform:
+    parent: "Send your traces to Checkly"
+    identifier: troubleshooting-missing-spans
+navTitle: Missing spans
+beta: true
+---
+
+This guide will walk you through the process of identifying missing spans, understanding their root causes, and implementing solutions to ensure your OpenTelemetry setup with Checkly provides complete and accurate tracing information. 
+
+## Missing spans
+
+Missing spans, also known as orphan spans, are segments of a trace that are disconnected from either their parent spans or from the root span.
+
+An instrumented distributed system, will emit spans creating a hierarchical chain that follows the flow of the request through the different components and services.
+
+When a span is missing, they create a whole in this chain, making it difficult to diagnose issue and understand the full picture.
```

Comment:
```suggestion
When a span is missing, they create a hole in this chain, making it difficult to diagnose issues and understand the full picture. If this happens, you will notice a span missing from the visualization in Checkly.
```

Reply:
Note: This is just a suggestion from my side, I feel like that'll help connect the dots, feel free to ignore if it doesn't make sense to you :)

Reply:
This is great, also thank you for catching the typo!